clients/src/main/resources/common/message/ControlledShutdownResponse.json

```diff
@@ -18,7 +18,10 @@
   "type": "response",
   "name": "ControlledShutdownResponse",
   // Versions 1 and 2 are the same as version 0.
-  "validVersions": "0-2",
+  //
+  // Version 3 is the first flexible version.
+  "validVersions": "0-3",
+  "flexibleVersions": "3+",
```

Comment:
Leaving the comment here because I haven't found a better place. It would be great if you could add tests in `RequestResponseTest#testSerialization` to cover all the versions which have been bumped.

Reply:
Hmm.  We don't generally do exhaustive testing of all versions in RequestResponseTest.  There would be a lot of entries!  But I agree we should add more stuff in RequestResponseTest.  Let's think about it in a follow-on PR